ui-components/src/blocks/ImageCard.tsx

```diff
@@ -26,7 +27,9 @@ const ImageCard = (props: ImageCardProps) => {
   const image = (
     <div className="image-card__wrapper">
       <figure className="image-card">
-        {props.imageUrl && <img src={props.imageUrl} alt={"A picture of the building"} />}
+        {props.imageUrl && (
+          <img src={props.imageUrl} alt={props.description || "An picture of the building"} />
```

Comment:
oops @seanmalbert I just had this PR open again because I was demoing it to the rest of our team, and I noticed that this default has a typo in it 😕